authentication/config.go

```diff
@@ -100,9 +101,9 @@ func (c Config) BuildOAuthConfig(activeDirectoryEndpoint string) (*OAuthConfig,
 	return &multiAuth, nil
 }
 
-// BearerAuthorizerCallback returns a BearerAuthorizer valid only for the Primary Tenant
+// ADALBearerAuthorizerCallback returns a BearerAuthorizer valid only for the Primary Tenant
```

Comment:
Open to suggestions on naming these and/or whether we can break this exported method?